docs/sql/statements/explain.rst

```diff
@@ -19,6 +19,7 @@ Synopsis
 ::
 
     EXPLAIN [ ANALYZE ] statement
+    EXPLAIN [ (ANALYZE | COST) (TRUE | FALSE) ] statement
```

Comment:
Doesn't this mean that we cannot do `explain analyze <stmt>` anymore?

Reply:
The synopsis can be split into 2 statements e.g.: https://github.com/crate/crate/blob/master/docs/sql/statements/set.rst#id7